sdk_container/src/third_party/coreos-overlay/sys-libs/nss-usrfiles/nss-usrfiles-9999.ebuild

```diff
@@ -2,18 +2,16 @@
 # Distributed under the terms of the GNU General Public License v2
 
 EAPI=7
-CROS_WORKON_PROJECT="flatcar/nss-altfiles"
-CROS_WORKON_LOCALNAME="nss-altfiles"
-CROS_WORKON_REPO="https://github.com"
+EGIT_REPO_URI="https://github.com/flatcar/nss-altfiles.git"
 
 if [[ "${PV}" == 9999 ]]; then
 	KEYWORDS="~amd64 ~arm ~arm64 ~x86"
 else
-	CROS_WORKON_COMMIT="c8e05a08a2e28eb48c6c788e3007d94f8d8de5cd" # flatcar-master
+	EGIT_COMMIT="c8e05a08a2e28eb48c6c788e3007d94f8d8de5cd" # flatcar-master
```

Comment:
That's because that repo's default branch on GitHub is master rather than flatcar-master. I can change that if you think it's okay. I've now checked all the others too, they are correct.

Reply:
Why is there the need to be branch bound?  A commit is a unique point in the history and it is in git terms, not dependent of a branch, it is the other way around.

Reply:
If this is a `git-r3` implementation detail, what happens when we try to do dev&test with our own repos and branches? It will not work unless we have the git commit on the flatcar-master branch in our own repo?

Reply:
It's because git-r3 only clones a single branch by default to save space. You can pick specify any commit in that branch. If you don't specify the branch, then it uses the repo's default.

With flatcar_workon, no specific commit is set by the live ebuild, so it will use whatever commit is currently checked out in the working repo. I think this makes the most sense. You can still override it by setting `EGIT_OVERRIDE_BRANCH_FLATCAR_AFTERBURN` or similar if you really want to.

Reply:
Then I think this new behaviour needs to be documented here, so that there is a way to point developers to how the new workflow works when needed (that CROS_WORKON_COMMIT changed to EGIT_COMMIT and that EGIT_OVERRIDE_BRANCH_COREOS_AFTERBURN should be used if their commit is on another branch) -> https://www.flatcar.org/docs/latest/reference/developer-guides/sdk-tips-and-tricks/#modify-or-update-individual-packages

Reply:
If it's a different branch for a versioned ebuild then you just set `EGIT_BRANCH`, and yes, I can see that you might want to do that. Thanks for the pointer, I'll update those docs.

Reply:
And a changelog would be nice with the constant rename. https://github.com/flatcar/scripts/tree/main/changelog/changes

Reply:
The changelog is used to generate release notes, and generally is about changes to the produced images, not the SDK.

Reply:
The usage of another branch is important, as the PRs to the cross repos should be done via a different branch, otherwise the closed PRs branch (if it is always flatcar-master) it is going to be hard reset and contextual history will be lost.

Reply:
> The changelog is used to generate release notes, and generally is about changes to the produced images, not the SDK.

then the docs should be enough. I was thinking that this might help developers to better pinpoint when this behaviour changed.